cogboard-webapp/package.json

```diff
@@ -11,6 +11,7 @@
     "http-proxy-middleware": "^0.20.0",
     "jwt-decode": "^2.2.0",
     "moment-timezone": "^0.5.26",
+    "ramda": "^0.26.1",
```

Comment:
what is this for?

Reply:
It is javascript functional utilities library: https://ramdajs.com/